flake.nix

```diff
@@ -40,9 +40,9 @@
     nix-unit.inputs.flake-parts.follows = "flake-parts";
     nix-unit.inputs.nixpkgs.follows = "nixpkgs";
     nix-unit.inputs.treefmt-nix.follows = "treefmt-nix";
-    ocaml-flake.url = "github:9glenda/ocaml-flake";
-    ocaml-flake.inputs.nixpkgs.follows = "nixpkgs";
-    ocaml-flake.inputs.treefmt-nix.follows = "treefmt-nix";
+    nixos-healthchecks.url = "git+file:///home/palo/dev/nixos/healthcheck";
```

Comment:
This will have to be the public URL.

You could use `nix build --override-input nixos-healthchecks git+file:///home/palo/dev/nixos/healthcheck` for local testing.

Reply:
:facepalm: yeah, ... I fixed it.